configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+echo "Running configuration script"
+export DEBIAN_FRONTEND=noninteractive
+
+echo "US/Central" | sudo tee /etc/timezone
+dpkg-reconfigure --frontend noninteractive tzdata
+
+echo "Install essential software pacakges"
+apt-get -qq update
+apt-get -qq install -y wget htop wget build-essential zip software-properties-common gnupg awscli
+apt-get -qq install -y ansible
+echo $PERM_ENV  > /data/www/host.txt
+
+echo "Add custom sources"
+# Add mysql key
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 5072E1F5
```

Comment:
What's going on here?  And if we're identifying a key, shouldn't we be using the long-form fingerprint?  (The short ones are easily faked, as the infamous "evil set" of GPG keys proved -- see https://evil32.com/ for details.)

Reply:
I'm gonna take these out at this point -- ansible handles the install of these kinds of dependencies.  I think I was trying to install very specific versions of these when I had included this.